src/Autosuggest.js

```diff
@@ -18,6 +20,10 @@ export default class Autosuggest extends Component { // eslint-disable-line no-s
     id: PropTypes.string                    // Used in aria-* attributes. If multiple Autosuggest's are rendered on a page, they must have unique ids.
   }
 
+  componentWillReceiveProps(nextProps) {
+    this.onUpdate(nextProps.value);
```

Comment:
This should be wrapped in an `if (typeof nextProps.value === 'string')` statement so it doesn't trigger if the developer is using the component in an uncontrolled way and they decide to change some other prop (like `inputAttributes.className`). I will update this in my next commit.